src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -633,6 +633,9 @@ internal async Task SendInvocationRequest(ScriptInvocationContext context)
                     context.CancellationToken.Register(() => SendInvocationCancel(invocationRequest.InvocationId));
                 }
 
+                _metricsLogger.LogEvent(string.Format(MetricEventNames.WorkerInvoked, Id), functionName: context.FunctionMetadata.Name);
+                _workerChannelLogger.LogInformation("Sending invocation request with invocationId: {invocationId} on workerId: {workerId} for function: {functionName}", invocationRequest.InvocationId, Id, context.FunctionMetadata.Name);
```

Comment:
I think we should remove these 2 new log statements. They're not needed for the new metrics you're logging, and will have a performance impact in high throughput scenarios. In general on the execution hotpath we want to minimize the per invocation logging we do since past performance work has shown it has a big impact.

Reply:
Done